src/api/todos.ts

```diff
@@ -0,0 +1,19 @@
+import { Todo } from '../types/Todo';
+import { NewTodo } from '../types/NewTodo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodo = (newTodo: NewTodo) => {
+  return client.post<Todo>('/todos', newTodo);
+};
+
+export const deleteTodo = (todoId: number) => {
+  return client.delete(`/todos/${todoId}`);
+};
+
+export const updateTodo = (todoId: number, changingPart: Partial<Todo>) => {
```

Comment:
Consider renaming 'changingPart' to a more descriptive name like 'updatedFields'.